tracking/speed_groups.cpp

```diff
@@ -0,0 +1,22 @@
+#include "tracking/speed_groups.hpp"
+
+namespace tracking
+{
+uint32_t const kSpeedGroupThresholdPercentage[] = {8, 16, 33, 58, 83, 100, 100, 100};
```

Comment:
I don't see how it helps: the ARRAY_SIZE is checked against the declaration in .hpp and such an assert looks tautological.